ui/src/components/device/DeviceDetails.vue

```diff
@@ -193,6 +211,89 @@ export default {
     receiveName(params) {
       this.device.name = params;
     },
+
+    findLastOne(newList) {
+      const difference = this.oldList.filter((x) => !newList.includes(x))
+        .concat(newList.filter((x) => !this.oldList.includes(x)));
+
+      return difference[0];
+    },
+
+    tagRule() {
+      if (this.errorMsg !== '') {
+        return this.errorMsg;
+      }
+
+      return true;
+    },
+
+    async actionTag(newList) {
+      const tag = this.findLastOne(newList);
+      const device = this.$store.getters['devices/get'];
+
+      const data = { uid: device.uid, tag };
+
+      if (this.oldList.length < newList.length) {
+        try {
+          this.errorMsg = '';
+          await this.$store.dispatch('devices/createTag', data);
+          this.$store.dispatch('snackbar/showSnackbarSuccessAction', this.$success.deviceTagCreate);
+          this.oldList = newList;
+          return 'false';
+        } catch (error) {
+          this.$nextTick(() => this.list.pop());
+          switch (true) {
+          // when the name the format is invalid.
+          case (error.response.status === 400): {
+            this.errorMsg = 'The format is invalid. Min 3, Max 255 characters!';
+            this.$store.dispatch('snackbar/showSnackbarErrorAction', this.$errors.snackbar.deviceTagCreate);
```

Comment:
When the error is defined in the component, it is not necessary to show the error in the snackbar.